Packs/SplunkPy/Integrations/SplunkPy/SplunkPy_description.md

```diff
@@ -2,12 +2,28 @@
 Use the SplunkPy integration to fetch incidents from Splunk ES, and query results by SID.
 ***
 
-To use Splunk token authentication, enter the text: *_token* in the **Username** field and your token value in the **Password** field.
+ - To use Splunk token authentication, enter the text: *_token* in the **Username** field and your token value in the **Password** field.
 To create an authentication token, go to [Splunk create authentication tokens](https://docs.splunk.com/Documentation/SplunkCloud/8.1.2101/Security/CreateAuthTokens).
+ - In case of inconsistent authentication issues when using username & passowrd, try to use **\<USERNAME\>@_basic** as username.
```

Comment:
```suggestion
 - In case of inconsistent authentication issues when using username & password, try to use **\<USERNAME\>@_basic** as the username.
```